config/locales/client.en.yml

```diff
@@ -4653,9 +4652,6 @@ en:
           headers: "Headers"
           payload: "Payload"
           body: "Body"
-          go_list: "Go to list"
-          go_details: "Edit webhook"
```

Comment:
The icon pencil button needs an "Edit" title attribute. Could be a generic "Edit" string though.